ibex_configs.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Ibex configurations files, holds the parameter sets that are tested under CI.
+# Each configuration must specify the same set of parameters
+
+# Two-stage pipeline without additional branch target ALU and 2 cycle multiplier
+# (3 cycles for mulh)
```

Comment:
I've messed up these cycle counts, it's 3 for mul and 4 for mulh.

Should also explain that stall cycles are actually one less (so single cycle mul means no stalling, 3 cycle mul means 2 cycles of stall on mul)